plugins/kubernetes/README.md

```diff
@@ -14,26 +14,10 @@ It is only meant for local development, and the setup for it can be found inside
 
 ## Surfacing your Kubernetes components as part of an entity
 
-There are two ways to surface your kubernetes components as part of an entity.
+There are 2 ways to surface your kubernetes components as part of an entity.
```

Comment:
why though :D haha, commonly you use the word form for small numbers

Reply:
haha yea was pulling in changes from my original approach, not purposely changing that back to a number